sig-network/charter.md

```diff
@@ -0,0 +1,46 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubeVirt networking.
+The main objective of the SIG is to connect the Virtual Machine to the primary and secondary pod networks,
```

Comment:
nit:
```suggestion
The main objective of the SIG is to connect the guest to cluster services including: pod network, secondary networks,
```

Reply:
I prefer the original. I wanted to emphasize the connection to the primary and secondary networks  is the main goad. Using the word "including" gives the feeling it is yet another service.

Reply:
@ormergi Can I resolve this comment or is it a blocker from your side?

Reply:
No blocker from my side, it a nit and I already approved this PR on previous review cycle :slightly_smiling_face: 
https://github.com/kubevirt/community/pull/271#pullrequestreview-1970215570